Dfe.PrepareConversions/Dfe.PrepareConversions/TagHelpers/KeyStage4DataTagHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿using DocumentFormat.OpenXml.Drawing;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Linq;
+
+namespace Dfe.PrepareConversions.TagHelpers
+{
+   public static class KeyStage4DataTagHelper
```

Comment:
I think TagHelpers generally inherit from TagHelper e.g. see - https://github.com/DFE-Digital/prepare-academy-conversions/blob/main/Dfe.PrepareConversions/Dfe.PrepareConversions/TagHelpers/BackLinkTagHelper.cs

Reply:
Yeah maybe this isn't really a taghelper, not sure where it would live. Perhaps a KeyStage4DataStatusService?

Reply:
I've moved it out and updated the syntax to be a bit nicer, maybe there's a better way still?